MiniSim/Views/Preferences.swift

```diff
@@ -11,8 +11,26 @@ import KeyboardShortcuts
 import LaunchAtLogin
 
 struct Preferences: View {
+    var menuImages = [ "menu_icon_1", "menu_icon_2", "menu_icon_3" ]
+    @State var menuImageSelected = 0
+    
     var body: some View {
         Settings.Container(contentWidth: 400) {
+            Settings.Section(title: "Icon:") {
+                Picker("", selection: $menuImageSelected) {
+                    ForEach(0 ..< menuImages.count, id: \.self) { image in
```

Comment:
This is an anti-pattern in SwiftUI to pass non static ranges like this. 

When you will create the enum with all images you can do `MenuIcons.allCases`